File: src/nl/hannahsten/texifyidea/LatexLanguage.java
package nl.hannahsten.texifyidea;

import com.intellij.lang.Language;

/**
 * @author Sten Wessel
 */
public class LatexLanguage extends Language {

    public static final LatexLanguage INSTANCE = new LatexLanguage();

    private LatexLanguage() {
        super("Latex");
    }
}

File: src/nl/hannahsten/texifyidea/grammar/LatexLanguage.kt
package nl.hannahsten.texifyidea.grammar

import com.intellij.lang.Language
import nl.hannahsten.texifyidea.TexifyBundle

/**
 * @author Sten Wessel
 */
object LatexLanguage : Language("Latex") {
    private fun readResolve(): Any = LatexLanguage

    override fun getDisplayName(): String = TexifyBundle.message("language.latex.displayName")
}
